docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci-basic.md

```diff
@@ -0,0 +1,137 @@
+   default credentials, the username and password are **admin** and **Harbor12345**, respectively.
+
+2. From the **Projects** section, click **New Project**. A project in Harbor contains all repositories of an
+   application.
+
+3. Give your project a name and keep the default settings for the remaining configuration. Click **OK** to proceed.
+
+4. In your terminal, export the `HARBOR_ADDRESS` variable, which will store your Harbor address. Do not include the
+   "https://" prefix.
+
+   ```bash
+   export HARBOR_ADDRESS=<Your_Harbor_Address>
+   ```
+
+5. Issue the command `oras login` to log in to your Harbor registry. When prompted, enter your username and password. If
+   the login is successful, you will receive a confirmation message.
+
+   ```bash
+   oras login $HARBOR_ADDRESS
+   ```
+
+6. Next, export the variables required for creating the Harbor repository and pushing the pack.
+
+   - `HARBOR_PROJECT` - Specify a name for the Harbor project that will store the repositories and the pack files. For
+     example, `spectro-oci-registry`.
+   - `NAME` - The pack's name, which must match the `name` parameter in the **pack.json** file.
+   - `VERSION` - The pack's version, which must match the `version` parameter in the **pack.json** file.
+
+   ```bash
+   export HARBOR_PROJECT=<Your_Repository_Name>
+   export NAME=<Your_Pack_Name>
+   export VERSION=<Your_Pack_Version>
+   ```
+
+7. Navigate to the directory containing the folder with the pack files.
+
+8. Before pushing the pack to the registry, compress the contents of the pack folder into an archive file. Issue the
+   command below to create the archive file. Replace `<Your_Pack_Folder_Name>` with the name of the folder containing
+   the pack files.
+
+   ```bash
+   tar -czvf $NAME-$VERSION.tar.gz <Your_Pack_Folder_Name>
+   ```
+
+9. Create a base path repository to store your pack repositories. Note that Harbor creates a repository when the user
+   pushes an artifact to a project.
+
+   ```bash
+   oras push $HARBOR_ADDRESS/$HARBOR_PROJECT/spectro-packs/archive
+   ```
+
+10. Now, proceed to create the pack repository and push your pack to the Harbor registry.
+
+    ```bash
+    oras push $HARBOR_ADDRESS/$HARBOR_PROJECT/spectro-packs/archive/$NAME:$VERSION $NAME-$VERSION.tar.gz
+    ```
+
+:::warning
+
+Make sure to include the **spectro-packs/archive** path in _all_ your repositories. Palette expects this namespace in
+custom OCI registries.
+
+:::
+
+11. After pushing the pack to the Harbor registry, follow the steps in [Add OCI Packs Registry](./add-oci-packs.md) to
+    add your Harbor registry to Palette.
+
+:::info
+
+Palette automatically synchronizes the registries. However, you can manually trigger the synchronization if needed. From
+the **OCI Registries** page, click the **three-dot Menu** next to the registry name you added and select **Sync**.
+
+:::
```

Comment:
I think you could indent it to the step 🤔

Reply:
:white_check_mark: